### PR TITLE
Pro 3120 widget preview

### PR DESCRIPTION
**Pull request by BoDonkey**
## Summary

*Summarize the changes briefly, including which issue/ticket this resolves. If it closes an existing Github issue, include "Closes #[issue number]"*

This PR modifies the `/guide/areas-and-widgets.md` and `/reference/field-types/area.md` sections of the documentation to add in the expanded widget preview menu information. Closes PRO-3120.

## What are the specific steps to test this change?

*For example:*
> 1. Run the website and log in as an admin
> 2. Open a piece manager modal and select several pieces
> 3. Click the "Archive" button on the top left of the manager and confirm that it should proceed
> 4. Check that all pieces have been archived properly

1. Build the documentation
2. Navigate to either `/guide/areas-and-widgets.html` or `reference/field-types/area.html`.

## What kind of change does this PR introduce?
*(Check at least one)*

- [ ] Bug fix
- [ ] New feature
- [ ] Refactor
- [x] Documentation
- [ ] Build-related changes
- [ ] Other

## Make sure the PR fulfills these requirements:

- [x] It includes a) the existing issue ID being resolved, b) a convincing reason for adding this feature, or c) a clear description of the bug it resolves
- [ ] The changelog is updated
- [ ] Related documentation has been updated
- [ ] Related tests have been updated

If adding a new feature without an already open issue, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

### CODE REVIEW

File: docs/guide/areas-and-widgets.md [hunk 1]
```diff
@@ -74,6 +82,88 @@ introduction: {
...
+  extend: '@apostrophecms/page-type',
+  options: {
+    label: 'Landing Page'
+  },
+  fields: {
+    add: {
+      main: {
+        type: 'area',
+        options: {
+      👉🏻  expanded: true,
```
Comment (boutell) [inline docs/guide/areas-and-widgets.md:103]: Nice use of emoji

Comment (boutell) [reply docs/guide/areas-and-widgets.md:103]: Make sure however that it lines up reasonably in "npm run dev" (i.e. in our actual vuepress build) which might not match the behavior of github markdown when it comes to fixed with fonts and emoji

File: docs/reference/field-types/area.md [hunk 2]
```diff
@@ -1,11 +1,13 @@
 # `area`
 
-An `area` field allows editors to add, edit, and arrange a series of [widgets](/reference/glossary.md#widget). The properties configured in `options` specify the allowed widget types and the configuration for those widgets.
+An `area` field allows editors to add, edit, and arrange a series of [widgets](/reference/glossary.md#widget). There are two ways to configure an `area`. The first adds widgets to a pop-up list menu for the editor selection. For this configuration, widgets are added through a `widgets` property in the `options`. Alternatively, the expanded preview menu provides a fly-in menu that allows grouping and a visual preview of widgets. This is configured through a `groups` option that takes the `widgets` property.
```
Comment (boutell) [inline docs/reference/field-types/area.md:3]: through a `groups` option in which each individual group takes a `widgets` property.

**Review by boutell — CHANGES_REQUESTED** [review]
Minor notes.

File: docs/guide/areas-and-widgets.md (hunk 1 above)
Comment (BoDonkey) [reply docs/guide/areas-and-widgets.md:103]: Checked and it looks good to me.

File: docs/reference/field-types/area.md (hunk 2 above)
Comment (BoDonkey) [reply docs/reference/field-types/area.md:3]: Changed.

**Review by boutell — APPROVED** [review]

**Review by boutell — APPROVED** [review]

**Pull request merged**